apps/src/blockly/customBlocks/googleBlockly/behaviorBlocks.js

```diff
@@ -196,7 +196,7 @@ export function flyoutCategory(workspace, functionEditorOpen = false) {
   };
 
   // If the modal function editor is enabled, we render a button to open the editor
-  // Otherwise, we render a "blank" behavior definition block
+  // Behaviors are not editable without the modal editor open
```

Comment:
I think this comment needed to be updated. Does that sound right?

Reply:
I think so, it looks like we don't actually create a blank block?

Reply:
Yeah, if we do, I don't see it in the code and I couldn't manually find a scenario in which we create one. I forgot that that was the case when I started this PR, though, and originally had code to support the case where behavior definitions would also be on the main workspace 🙃